.env.example

```diff
@@ -72,3 +72,4 @@ MIX_ROOM_NAME_LIMIT=50
 
 DEFAULT_LOGO=/images/logo.svg
 DEFAULT_ROOM_LIMIT=-1
+DEFAULT_PAGE_SIZE=15
```

Comment:
@dennis95stumm @rhrt04 We should agree on one setting name, see https://github.com/THM-Health/PILOS/pull/23/files#diff-cbfd64a28982a1818f2b5f16e7f9d634R76

Reply:
@rhrt04 I would prefer `DEFAULT_PAGINATION_PAGE_SIZE` but with the value 15, since 10 is too small i think.